api/alerting/alerting_controller.go

```diff
@@ -142,19 +145,25 @@ func UpdateEnvironmentAlertingConfig(accounts models.Accounts, w http.ResponseWr
 
 	var updateAlertingConfig alertingModels.UpdateAlertingConfig
 	if err := json.NewDecoder(r.Body).Decode(&updateAlertingConfig); err != nil {
-		radixhttp.ErrorResponse(w, r, err)
+		if err = radixhttp.ErrorResponse(w, r, err); err != nil {
+			log.Errorf("%s: failed to write response: %s", r.URL.Path, err.Error())
```

Comment:
Can we put this logging to the method `radixhttp.ErrorResponse` to avoid repetition?
`%v` can be used for error
`log.Errorf("%s: failed to write response: %v", r.URL.Path, err)`